cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/terminology/HashedCode.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * (C) Copyright IBM Corp. 2021, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.cohort.cql.terminology;
+
+import java.util.Objects;
+
+import org.opencds.cqf.cql.engine.runtime.Code;
+
+
+/**
+ * This class is a simple wrapper class providing equals and hashCode functions
+ * for the org.opencds.cqf.cql.engine.runtime.Code class
+ * This is used to enable Set lookups for codes within ValueSets, which will be
+ * more performant than iterating through a list of codes
+ *
+ */
+public class HashedCode {
```

Comment:
Did you look at our `CodeKey` class at all? It only uses the code and system for comparisons, but I wasn't sure if that's sufficient for the indexing you are hoping to add.

Reply:
See my note above to Corey on this topic.